modules/nw-route-specific-annotations.adoc

```diff
@@ -16,95 +16,130 @@ To create an allow list with multiple source IPs or subnets, use a space-delimit
+
+The maximum number of IP addresses and CIDR ranges directly visible in the `haproxy.config` file is 61. [^1^] 
+|
+
+|`haproxy.router.openshift.io/hsts_header` 
+|Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. 
+|
+
+|`haproxy.router.openshift.io/rewrite-target` 
+|Sets the rewrite path of the request on the backend. 
+|
+
+|`router.openshift.io/cookie-same-site` 
+|Sets a value to restrict cookies. The values are:
 
 `Lax`: the browser does not send cookies on cross-site requests, but does send cookies when users navigate to the origin site from an external site. This is the default browser behavior when the `SameSite` value is not specified.
 
 `Strict`: the browser sends cookies only for same-site requests.
 
 `None`: the browser sends cookies for both cross-site and same-site requests.
 
-This value is applicable to re-encrypt and edge routes only. For more information, see the link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite[SameSite cookies documentation].|
+This value is applicable to re-encrypt and edge routes only. For more information, see the link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite[SameSite cookies documentation].
+|
 
-|`haproxy.router.openshift.io/set-forwarded-headers` | Sets the policy for handling the `Forwarded` and `X-Forwarded-For` HTTP headers per route. The values are:
+|`haproxy.router.openshift.io/set-forwarded-headers` 
+|Sets the policy for handling the `Forwarded` and `X-Forwarded-For` HTTP headers per route. The values are:
 
 `append`: appends the header, preserving any existing header. This is the default value.
 
 `replace`: sets the header, removing any existing header.
 
 `never`: never sets the header, but preserves any existing header.
 
-`if-none`: sets the header if it is not already set.| `ROUTER_SET_FORWARDED_HEADERS`
+`if-none`: sets the header if it is not already set.
+|`spec.httpHeaders.forwardedHeaderPolicy` 
 
 |===
 [.small]
---
-1. By default, the router reloads every 5 s which resets the balancing connection across pods from the beginning. As a result, the `roundrobin` state is not preserved across reloads. This algorithm works best when pods have nearly identical computing capabilites and storage capacity. If your application or service has continuously changing endpoints, for example, due to the use of a CI/CD pipeline, uneven balancing can result. In this case, use a different algorithm.
-2. If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from the `haproxy.config` file. This file is stored in the `/var/lib/haproxy/router/allowlists` folder.
+
+1. If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from `haproxy.config`. This file is stored in the `var/lib/haproxy/router/whitelists` folder.
```

Comment:
it looks like whitelist has been changed to allowlist, so `var/lib/haproxy/router/whitelists` should be changed to `/var/lib/haproxy/router/allowlists`